woocommerce/woocommerce
# Configured wp-env to work with Playwright tests

### Changes proposed in this Pull Request:

This PR configures the test environment created by `wp-env` to reliably run Playwright tests while still being able to run against the `e2e-environment` package.

The main reasons for these changes are: 
- to start the process of moving away from the `@woocommerce/e2e-environment` and `@woocommerce/e2e-environment` packages in favor of the `wp-env` package when running Playwight tests. 
- Using the same environment tool that is used to develop the WooCommerce plugin.
- `wp-env` is way more configurable than its predecessor.

**Note**
I tried to reduce changes to the tests as much as possible. The changes that do exist were necessary to get test to consistently pass locally.

### How to test the changes in this Pull Request:
 **Test against wp-env**

1. Switch this branch 
2. Run `pnpm env:test --filter=woocommerce` to spin and initialize up the test environment.
3. Go into the woocommerce plugin directory using `cd plugins/woocommerce`
4. Execute PW tests using `USE_WP_ENV=1 pnpm playwright test --config=e2e/playwright.config.js`
5. Observe that no tests fail.

**Test against e2e-environment**
1. Switch this branch 
2. Run `pnpm docker:up --filter=woocommerce` to spin and initialize up the test environment.
3. Go into the woocommerce plugin directory using `cd plugins/woocommerce`
4. Execute PW tests using `pnpm playwright test --config=e2e/playwright.config.js`
5. Observe that no tests fail.

### Other information:




### FOR PR REVIEWER ONLY:
